### PR TITLE
Update MVEWrapper and test to keras_core

### PR DESCRIPTION
**Pull request by vballoli**

### CODE REVIEW

**Comment by vballoli** [comment]
Reference issue: https://github.com/themis-ai/capsa-core/issues/3

File: capsa/aleatoric/mve.py [hunk 1]
```diff
@@ -1,24 +1,24 @@
-import tensorflow as tf
+import keras_core as keras
 
 from ..base_wrapper import BaseWrapper
 from ..risk_tensor import RiskTensor
```
Comment (egesko) [inline capsa/aleatoric/mve.py:4]: This should cause cause capsa-core to load tensorflow. Any chance you can comment this?

File: test/test_mve.py [hunk 2]
```diff
@@ -1,8 +1,7 @@
 import unittest
 import numpy as np
-import tensorflow as tf
+import keras_core as keras
 
-from tensorflow import keras
 import tensorflow_probability as tfp
```
Comment (egesko) [inline test/test_mve.py:5]: We should not need `tensorflow_probability` here as well.

**Review by egesko — CHANGES_REQUESTED** [review]
I left couple of comments, but overall it looks great. Thank you!

File: test/test_mve.py (hunk 2 above)
Comment (vballoli) [reply test/test_mve.py:5]: `tfp` was required for the test data generation. Should we just shift to having them implemented from scratch using the `keras_core.ops` for `MultivariateNormalDiag` (moon data)? (and a small implementation of the quantiles for normal distribution - calib plot )

File: capsa/aleatoric/mve.py (hunk 1 above)
Comment (vballoli) [reply capsa/aleatoric/mve.py:4]: Not sure I got this: tf is not being imported right? so keras_core should ideally load whatever is in the environment variable?

Comment (egesko) [reply capsa/aleatoric/mve.py:4]: Oh, my mistake. I read the change incorrectly.

File: test/test_mve.py (hunk 2 above)
Comment (egesko) [reply test/test_mve.py:5]: Yeah, I think that's the best approach. Ideally, we won't to remove all mentions of `tf`,`tfp`. But we can implement that in a different PR.

**Review by egesko — APPROVED** [review]
LGTM! Thank you again!

**Pull request merged**